### PR TITLE
Remove react

### PR DESCRIPTION
**Pull request by marclave**

### CODE REVIEW

File: packages/column-extension/src/index.ts [hunk 1]
```diff
@@ -1,31 +1,28 @@
...
 
-    return extensions;
+    return extensions
   },
-});
+})
 
-export { Column, ColumnBlock };
-
-export default ColumnExtension;
```
Comment (leonard-henriquez) [inline packages/column-extension/src/index.ts]: Removing the default export introduces a breaking change

**Review by leonard-henriquez — COMMENTED** [review]
Thank you Marc for this PR ! 😍 
Could you enforce the prettier convention of the repo?
Can you also add a description to the PR to allow other people to review it?
I made also a comment on a default export being deleted
Thank you in advance!

**Comment by marclave** [comment]
@leonard-henriquez thanks for the feedback! I will get this in tonight :)

**Comment by marclave** [comment]
@leonard-henriquez wow, the week totally just blew by!

I added a lint command :) and I think i got your comments done

**Review by leonard-henriquez — APPROVED** [review]
Thank you @marclave !!

**Pull request merged**